libr/anal/fcn.c

```diff
@@ -1171,15 +1215,15 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 		// Case of valid but unused "add [rax], al"
 		case R_ANAL_OP_TYPE_ADD:
 			if (anal->opt.ijmp) {
-				if (((op.size + 4) <= len) && !memcmp (buf + op.size, "\x00\x00\x00\x00", 4)) {
+				if (!memcmp (buf + op.size, "\x00\x00\x00\x00", 4)) {
```

Comment:
risky... I'd keep the check because if `op.size + 4 >= 32` you would read out of bounds.